poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -1431,12 +1443,12 @@ private int findStartOfRowOutlineGroup(int rowIndex) {
 
     private int writeHidden(SXSSFRow xRow, int rowIndex) {
         int level = xRow.getOutlineLevel();
-        SXSSFRow currRow = getRow(rowIndex);
+        SXSSFRow currRow = (SXSSFRow) getRow(rowIndex);
```

Comment:
Now that you've changed getRow to return `Row` - how can we just cast to SXSSFRow? Couldn't this now also be an XSSFRow?

Reply:
Agree, actually the reference to SXSSFRow is not needed here, instead Row would be fine.